src/app/workflow/versions/versions.component.html

```diff
@@ -118,6 +118,24 @@
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="engineVersions">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="The engine version used by the version's descriptor file(s)."
+        matTooltipPosition="above"
+      >
+        Engine Versions
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <div *ngFor="let engineVersion of version.versionMetadata?.engineVersions">
+          <div class="no-wrap">{{ engineVersion }}</div>
```

Comment:
Made this `no-wrap` otherwise it wraps the nextflow engine version in an unfriendly way like below

![Screenshot 2024-08-07 at 11-35-01 Dockstore Workflow github com_nf-core_rnaseq](https://github.com/user-attachments/assets/24f277a0-663f-4bd4-90af-32db7f5381cd)

When there are multiple engine versions, they are displayed one per line

Reply:
To avoid the repetition and because we're used to seeing the most significant part of a phrase on the left of text (in left-to-right languages), I wonder if we should strip `Nextflow` for the version column values, e.g., just show `!>=22.10.1` instead of `Nextflow !>=22.10.1`. It would also make the table a little less crowded.

I think it still makes sense to have the full string, with the `Nextflow` word on the facets page, as there are all types of workflows on the search page, whereas for this table the column is only visible on a Nextflow workflow.

Reply:
Seems intentional that we're storing the engine name based on this comment 

https://github.com/dockstore/dockstore/blob/ee8503a70285afdc8fcfdb70580c59cdf8130889/dockstore-webservice/src/main/java/io/dockstore/webservice/languages/NextflowHandler.java#L492-L494

Proposing we leave it as is for now and create a new ticket if it becomes an issue since it'll require a webservice change

Reply:
Lol, citing my own comment against my argument. :) 

I'd still argue that it's redundant on this page for Nextflow. My suggestion, which wasn't clear, would be to have a pipe here, not to change it on the backend -- I think it still makes sense to have `Nextflow !>=22.10.1` on the facets page for differentiation when we add other engines for other languages. But here it's already a Nextflow workflow, and there are no other engines for Nextflow other than Nextflow itself, so it's not really adding anything and slightly getting in the way of viewing the significant info.